examples/frontend_language/usage/srt_engine.py

```diff
@@ -0,0 +1,20 @@
+import sglang as sgl
+
+# Sample prompts.
```

Comment:
The code needs to be wrapped in `if __name__ == "__main__"` to make it work. Because the engine use `spawn` to create subprocesses.
Please add this hint into the example.

Reply:
thanks for catching this!!